airflow/api_connexion/endpoints/task_instance_endpoint.py

```diff
@@ -73,9 +74,22 @@ def get_task_instance(dag_id: str, dag_run_id: str, task_id: str, session=None):
     return task_instance_schema.dump(task_instance)
 
 
+def _convert_state(states):
+    if not states:
+        return
+    for idx, item in enumerate(states):
+        # we do not support 'null' maybe we should?
+        # to support null, we should add it to TaskState in
+        # openapi doc
```

Comment:
Why though? The state is called `NONE` and the Python counterpart is `None`. I don’t think we use `null` anywhere in the public API.

Reply:
I will then remove it